.eslintrc.js

```diff
@@ -10,17 +10,17 @@ module.exports = {
     sourceType: 'module',
   },
   plugins: ['@typescript-eslint', 'prettier'],
-  rules: {
-    'import/extensions': 'off',
-    'no-undef': 'off',
-    'consistent-return': 'off',
-    'prettier/prettier': ['error', {}, { usePrettierrc: true }],
-    'no-underscore-dangle': 'off',
-    'no-shadow': 'off',
-    'no-unused-vars': 'off',
-    'object-curly-newline': 'off',
-    'import/no-named-as-default': 'off',
-    'no-use-before-define': 'off',
-    'operator-linebreak': 'off',
-  },
+  // rules: {
```

Comment:
If its up and running uncomment out the rules for your push

Reply:
Done!